dashboard/lib/services/curriculum_pdfs/resources.rb

```diff
@@ -113,21 +113,24 @@ def generate_lesson_resources_title_page(lesson, directory="/tmp/")
 
           # we've been having some issues with these title page PDFs not
           # existing on the filesystem when it comes time to make the rollup.
+          #
           # It's not yet clear whether that's because this step is failing to
-          # generate or because the file is getting vanished after generation.
-          # Adding some logging here to help diagnose.
-          unless $?.success?
-            ChatClient.log(
-              "PDF generation exited with status code #{$?.exitstatus.inspect}",
```

Comment:
Did you get any helpful error information here? If so, can you summarize what it was before removing it?

Reply:
Yes and no. This clause never triggered at any point, so in once sense I got no information, and in another sense I learned that even when we end up failing to generate a file (as indicated by the other clause triggering), the process still _reports_ a success. So I'm removing the clause, but it is worth keeping in mind that whatever's going on here, it's subtle.

Reply:
That's what I figured, but wanted to double check. Thanks!